config/manifests/aws-efs-csi-driver-operator.package.yaml

```diff
@@ -1,4 +1,4 @@
 packageName: aws-efs-csi-driver-operator
 channels:
-- name: "4.11"
+- name: stable
   currentCSV: aws-efs-csi-driver-operator.v4.11.0
```

Comment:
Do you know we still have to carry versioning in CSV name? is that so that one can find out which CSV version they have it installed?

Reply:
It should be possible to change it if we also update it here:
https://github.com/openshift/aws-efs-csi-driver-operator/blob/e8b155c629369d20033cc452f2c12362977446a5/config/manifests/stable/aws-efs-csi-driver-operator.clusterserviceversion.yaml#L4
But yeah I thought it would be useful to easily see which CSV version is installed, and we'll still have to update the other instances of 4.x in the CSV each release in any case.